web/src/components/jobs/JobDetailPage.tsx

```diff
@@ -30,23 +33,35 @@ const styles = ({ spacing }: ITheme) => {
   return createStyles({
     root: {
       padding: spacing(2)
+    },
+    buttonDelete: {
+      backgroundColor: theme.palette.error.main,
+      color: 'white',
```

Comment:
I think I like doing the `theme.palette.common.white` for these in case we ever wish to support light mode.

Reply:
Sure, added it